froide/comments/forms.py

```diff
@@ -18,3 +18,16 @@ class CommentForm(DjangoCommentForm):
         widget=forms.Textarea(attrs={"class": "form-control", "rows": "4"}),
         max_length=COMMENT_MAX_LENGTH,
     )
+    is_moderation = forms.BooleanField(
+        widget=forms.CheckboxInput(attrs={"class": "form-check-input"}),
+        required=False,
+        initial=True,
+        label=_("This is a moderation comment"),
+    )
+
+    def __init__(self, target_object, data=None, initial=None, **kwargs):
+        super().__init__(target_object, data=data, initial=initial, **kwargs)
+        object_id = self.target_object.id
+        for field in self.fields:
+            id = "id_{}_{}".format(field, object_id)
+            self.fields[field].widget.attrs["id"] = id
```

Comment:
Unsure if this is the way to go. Do [form prefixes](https://docs.djangoproject.com/en/4.0/ref/forms/api/#prefixes-for-forms) not work here? They would be cleaner but could be that they somehow don't work here.

Reply:
Cool, did not know about form prefixes, tried ist, but prefixes change the name of the field and the [post view](https://github.com/django/django-contrib-comments/blob/master/django_comments/views/comments.py#L49) of django-comments can't deal with that.